pkg/crinit/aggregated/init.go

```diff
@@ -111,15 +122,29 @@ func newSubCmdInit(cmdOut io.Writer, pathOptions *clientcmd.PathOptions,
 }
 
 // validateOptions ensures that options are valid.
-func validateOptions(opts *aggregatedClusterRegistryOptions) error {
+func validateOptions(opts *aggregatedClusterRegistryOptions, cmdOut io.Writer) error {
 	opts.APIServerServiceType = v1.ServiceType(opts.apiServerServiceTypeString)
+	if len(opts.AggContext) == 0 {
+		glog.Warningf(
+			"Using context %q for aggregator since no dedicated context specified", opts.Host)
+		fmt.Fprintln(cmdOut, "WARNING: Using context %q for aggregator since no dedicated context specified")
+		opts.AggContext = opts.Host
```

Comment:
I think this should be moved elsewhere: this isn't a validation step, but rather a defaulting step.